dinput_hook/renderer_hook.cpp

```diff
@@ -417,10 +417,18 @@ void debug_render_mesh(const swrModel_Mesh *mesh, int light_index, int num_enabl
     static std::vector<Vertex> triangles;
     parse_display_list_commands(model_matrix, mesh, triangles);
 
-    glVertexAttribPointer(0, 3, GL_FLOAT, GL_FALSE, sizeof(triangles[0]), &triangles[0].pos);
-    glVertexAttribPointer(1, 4, GL_FLOAT, GL_FALSE, sizeof(triangles[0]), &triangles[0].color);
-    glVertexAttribPointer(2, 2, GL_SHORT, GL_FALSE, sizeof(triangles[0]), &triangles[0].tu);
-    glVertexAttribPointer(3, 3, GL_FLOAT, GL_FALSE, sizeof(triangles[0]), &triangles[0].normal);
+    glBindBuffer(GL_ARRAY_BUFFER, shader.VBO);
```

Comment:
i think it would be best to cache buffers per `swrModel_Mesh*` instead, that would be faster

Reply:
Instead of recreating a buffer each time ?
I would like to move to a model rendering from our side using only the `MODEL_ID`. Then we get the ID of the model the game wants to draw and we draw it like we want to.

I don't want to change it because I'm lazy and I don't think it's that performance sensitive yet ? (Make it work, then make it fast). This code will have to go later for sure though.